Branch/BNCNetworkAPIService.m

```diff
@@ -147,6 +147,7 @@ - (void) appendV2APIParametersWithDictionary:(NSMutableDictionary*)dictionary {
         userData[@"limit_facebook_tracking"] = BNCWireFormatFromBool(self.settings.limitFacebookTracking);
         userData[@"sdk"] = @"mac";
         userData[@"sdk_version"] = Branch.kitDisplayVersion;
+        userData[@"identity"] = self.settings.userIdentityForDeveloper;
```

Comment:
For v2/event endpoints, the API actually expects it for the key 'developer_identity'

Reply:
> For v2/event endpoints, the API actually expects it for the key 'developer_identity'

Ah ok, just updated it.